tests/runner.py

```diff
@@ -68,6 +68,21 @@ def path_from_root(*pathelems):
 
   python tests/runner.py random50
 
+An individual test can be skipped by passing the "skip:" prefix. E.g.
```

Comment:
these docs are also at http://kripken.github.io/emscripten-site/docs/getting_started/test-suite.html?highlight=testing

so we should update that location as well.

another option is to remove the docs here and make them point to the site?